scalafix-testkit/src/main/scala/scalafix/testkit/RuleTest.scala

```diff
@@ -2,8 +2,7 @@ package scalafix.testkit
 
 import scala.meta._
 import scala.meta.internal.symtab.SymbolTable
-
-import metaconfig.Conf
+import metaconfig.{Conf, Configured}
```

Comment:
make sure to run the [formatters](https://github.com/scalacenter/scalafix/blob/main/CONTRIBUTING.md#formatting) to please the CI

Reply:
Done